src/components/data-exploration/differential-expression-tool/LaunchPathwayAnalysisModal.jsx

```diff
@@ -55,6 +55,7 @@ const LaunchPathwayAnalysisModal = (props) => {
         >
           {/* display the alert only if there are no filter applied to diff expr */}
           <Paragraph style={{ width: '100%' }}>
+            {!advancedFiltersAdded && (
```

Comment:
Could you make `<Paragraph` and the `<Row` that holds this `<Alert` also not show up if `advancedFiltersAdded`? Or does it look ugly if not?